index.js

```diff
@@ -50,7 +58,21 @@
 // // => "different"
 // console.log("%c----------", "color: red") 
 
-
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum) {
+    console.log("same same")
+  }
+  else if (num == maybeNum) {
+    console.log("same same (but different)")
+  }
+  else {
+    console.log("different")
+  }
+}
```

Comment:
perfect.  Another option for line 65 would be to use parseInt to convert maybeNum to an integer and then use ===

if you want to dig a little deeper you can use typeof to determine the data type
e.g.
if (typeof num === 'number' && typeof maybeNum === 'number' && num === maybeNum)